components/event-calendar.vue

```diff
@@ -132,13 +91,30 @@ export default {
     }
   },
   methods: {
-    eventGroup(event) {
-      return groupForEvent(event, this.$store.state.groups.all)
-    },
     eventsOnDay(day) {
-      return this.events.filter(event =>
+      // get the events on this day
+      var events = this.events.filter(event =>
         isSameDay(new Date(event.startTime), day)
       )
+
+      var results = events.reduce(function(acc, obj) {
+        var key = obj['group']
+
+        if (!acc[key]) {
+          acc[key] = []
+        }
+
+        acc[key].push(obj)
+        return acc
+      }, {})
+
+      return results
+    },
+    groupsFirstEventOnDay(groupEvents) {
+      return groupEvents[0]
+    },
+    otherEventsForGroupOnDay(groupEvents) {
+      return groupEvents.slice(1, groupEvents.length)
```

Comment:
What do you think about naming these `firstEvent` and `otherEvents`?

I think the consistent argument name and use of both functions makes it clear that an array of events for a group on a particular day is to be provided and doesn't need to be called out in the function name.

Reply:
Works for me! I wasn't certain what to name these